data/campaigns/Winds_of_Fate/scenarios/05_The_Three_Sisters.cfg

```diff
@@ -0,0 +1,840 @@
+        {GENERIC_UNIT 1 (Water Serpent) ({X}) ({Y})}
+        [message]
+            speaker=Oxepotex
+            image_pos=right
+            mirror=yes
+            message= _ "I am Oxepotex!  Massster tamer of beassstsss big and sssmall." # no spellcheck
+        [/message]
+        [message]
+            speaker=Arinexis
+            message= _ "Sssettle down, Oxe.
+
+And get to work feeding your petsss!" # no spellcheck
+        [/message]
+    [/event]
+#enddef
+
+#define TEMP_SAURIAN_2 X Y
+    [event]
+        name=moveto
+        [filter]
+            side=1
+            x,y={X},{Y}
+        [/filter]
+        {GENERIC_UNIT 1 (Saurian Augur) ({X}) ({Y})}
+        {GENERIC_UNIT 1 (Saurian Augur) ({X}) ({Y})}
+    [/event]
+#enddef
+
+#define TEMP_MONSTERS X Y
+    [event]
+        name=moveto
+        [filter]
+            side=1
+            x,y={X},{Y}
+        [/filter]
+        [message]
+            speaker=unit
+            message= _ "I hear..."
+        [/message]
+        {NAMED_UNIT 4 (Giant Rat) ({X}) ({Y}) Tidbit "Tidbit" ()}
+        {NAMED_UNIT 4 (Giant Rat) ({X}) ({Y}) Morsel "Morsel" ()}
+        [message]
+            speaker=unit
+            message= _ "Vermin!"
+        [/message]
+        [message]
+            speaker=Tidbit
+            message= _ "<i>Reeeeee!</i>"
+        [/message]
+        [message]
+            speaker=unit
+            message= _ "I shall..."
+        [/message]
+        [message]
+            speaker=unit
+            [option]
+                description= _ "eat them."
+                [command]
+                    [store_unit_type]
+                        type=Giant Rat
+                        variable=monster
+                    [/store_unit_type]
+                    [harm_unit]
+                        [filter]
+                            type=Giant Rat
+                            side=4
+                        [/filter]
+                        [filter_second]
+                            id=$unit.id
+                        [/filter_second]
+                        amount=$monster.hitpoints
+                        fire_event=yes
+                        animate=yes
+                        experience=yes
+                    [/harm_unit]
+                    {CLEAR_VARIABLE monster}
+                [/command]
+            [/option]
+            [option]
+                description= _ "pet them."
+                [command]
+                    [message]
+                        speaker=Tidbit
+                        message= _ "<i>squeak</i>"
+                    [/message]
+                    [modify_unit]
+                        [filter]
+                            type=Giant Rat
+                        [/filter]
+                        side=1
+                    [/modify_unit]
+                    [message]
+                        speaker=narrator
+                        message= _ "Achievement Unlocked:
+
+<b>Leader of Mice</b>"
```

Comment:
I assume this was a joke - it made me laugh, at least...

Reply:
:p